src/main/java/com/ibm/cloudant/kafka/connect/CloudantSourceTask.java

```diff
@@ -211,27 +205,24 @@ public void start(Map<String, String> props) {
 
     @Override
     public void stop() {
+        // TODO needs a custom changes feed
```

Comment:
We could potentially store the running request in the task and call cancel on it if stop is called.
I don't think `changes().stop()` even went that far in the old client though.